misc/identifier-util/src/main/java/module-info.java

```diff
@@ -4,5 +4,5 @@
 
     exports io.ballerina.identifier to io.ballerina.lang, io.ballerina.runtime, io.ballerina.shell,
             io.ballerina.testerina.runtime, io.ballerina.lang.runtime, io.ballerina.lang.error,
-            ballerina.debug.adapter.core, io.ballerina.jsonmapper;
+            ballerina.debug.adapter.core, io.ballerina.jsonmapper, io.ballerina.cli;
```

Comment:
Why do we need to expose `io.ballerina.cli`?

Reply:
This is for the NativeUtils class. We are resolving the class name of different test classes to make the native image configs